Data/src/main/java/org/tribuo/data/columnar/extractors/DateExtractor.java

```diff
@@ -77,8 +102,14 @@ public DateExtractor(String fieldName, String metadataName, DateTimeFormatter fo
     @Override
     public void postConfig() {
         super.postConfig();
+        Locale locale;
+        if ((localeLanguage == null) && (localeCountry == null)) {
+            locale = Locale.getDefault(Locale.Category.FORMAT);
+        } else {
+            locale = new Locale(localeLanguage,localeCountry);
```

Comment:
Won't we end up here if one of `localeLanguage` and `localeCountry` are null?
Maybe if either one is null we use the default and log a warning.

Reply:
Changed to throw `PropertyException`.